<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="filenameLbl">
          <property name="text">
           <string>Имя файла JSON:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="filenameEdt">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="openFileBtn">
          <property name="text">
           <string>Выбрать файл...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="jsonFileLbl">
        <property name="text">
         <string>Содержимое файла:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="jsonFileText">
        <property name="font">
         <font>
          <family>Courier New</family>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="parseInfoLbl">
        <property name="text">
         <string>Результат проверки:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="parseInfoTxt">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="exitBtn">
          <property name="text">
           <string>Выход</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
